packages/fast-components-react-base/src/context-menu-item/context-menu-item.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { get } from "lodash-es";
+import { ContextMenuItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    ContextMenuItemHandledProps,
+    ContextMenuItemManagedClasses,
+    ContextMenuItemProps,
+    ContextMenuItemUnhandledProps,
+} from "./context-menu-item.props";
+
+export enum ContextMenuItemRole {
+    menuitem = "menuitem",
+    menuitemradio = "menuitemradio",
+    menuitemcheckbox = "menuitemcheckbox",
+}
+
+class ContextMenuItem extends Foundation<
+    ContextMenuItemHandledProps,
+    ContextMenuItemUnhandledProps,
+    {}
+> {
+    public static displayName: string = "ContextMenuItem";
+
+    public static defaultProps: Partial<ContextMenuItemProps> = {
+        role: ContextMenuItemRole.menuitem,
+        disabled: false,
+    };
+
+    protected handledProps: HandledProps<ContextMenuItemHandledProps> = {
+        managedClasses: void 0,
+        children: void 0,
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLDivElement> {
+        return (
+            <div
+                {...this.unhandledProps()}
+                className={this.generateClassNames()}
+                role={this.props.role}
+                aria-disabled={this.props.disabled || undefined}

```

Comment:
won't this never be undefined since a default has been set?

Reply:
not quite.

If `this.props.disabled` is false, it sets `aria-disabled` to `undefined`. This means no attribute shows up in the DOM.

If I just assigned it the value of `this.props.disabled`, then anything that is not disabled would get `aria-disabled="false"`, which seemed like a lot of useless clutter.